### PR TITLE
Python: add Payload and TransactionOptions

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues

#129 

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How the change has been tested

Running the new example

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/python/iota_sdk/wallet/common.py [hunk 1]
```diff
@@ -2,17 +2,53 @@
...
+            if isinstance(obj, (list, tuple, set)):
+                return type(obj)(remove_none(x) for x in obj if x is not None)
+            elif isinstance(obj, dict):
+                return type(obj)((remove_none(k), remove_none(v))
+                                 for k, v in obj.items() if k is not None and v is not None)
+            else:
+                return obj
+        deserialized_null_filtered = remove_none(deserialized)
+
+        message = dumps(humps.camelize(deserialized_null_filtered))
```
Comment (kwek20) [inline bindings/python/iota_sdk/wallet/common.py:51]: humps.camelize shouldnt be needed right?

Comment (Thoralf-M) [reply bindings/python/iota_sdk/wallet/common.py:51]: It is, unless we change the as_dict() method of `TransactionOptions`
Tbh it's all a bit messy with the serialization, for many other classes it's currently done in their as_dict() method which is a lot more code, but not sure what would be the best way for the whole project

Comment (kwek20) [reply bindings/python/iota_sdk/wallet/common.py:51]: hmmm true, thats a shame. Problem for another day though :D

**Review by kwek20 — APPROVED** [review]

File: bindings/python/iota_sdk/wallet/common.py (hunk 1 above)
Comment (Thoralf-M) [reply bindings/python/iota_sdk/wallet/common.py:51]: Opened an issue https://github.com/iotaledger/iota-sdk/issues/650

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**